man/fcase.Rd

```diff
@@ -18,42 +18,41 @@
   \code{\link{fifelse}}
 }
 \examples{
-x = 1:10 
+x = 1:10
 data.table::fcase(
 	x < 5L, 1L,
 	x > 5L, 3L
 )
-# [1]  1  1  1  1 NA  3  3  3  3  3
 
 data.table::fcase(
```

Comment:
no need `data.table::` prefix in examples, AFAIR we don't use it anywhere, and when running data.table example we can safely assume that pkg is attached.